halo2_proofs/tests/frontend_backend_split.rs

```diff
@@ -639,4 +619,9 @@ fn test_mycircuit_full_split() {
     )
     .expect("verify succeeds");
     println!("Verify: {:?}", start.elapsed());
+
+    assert_test_proof(
+        "e612e9d5ebcdcff5ed9823c527218cdbaaf9fe76cc5e42d05a60f652dcc92660",
```

Comment:
It would be nice to see that the proof in `test_mycircuit_full_split` is the same as in `test_mycircuit_full_legacy` because they come from the same circuit with same inputs.
I wonder why they are different :thinking:

Reply:
I found it!  The first test uses compressed selectors (the legacy API compresses selectors by default).  The split test uses uncompressed selectors.

Can you set `compress_selectors` to `true` in line 557?  Then the generated proof should be equal.